### ISSUE TITLE
Request: Movable Focus bar

### ISSUE THREAD

**Issue by Nmontalto** [cogwerkz/DiabolicUI#93]
**Description**
It would be nice to be able to freely move the focus bar as you could in the default UI, and if the focus bar had buffs/debuffs on it. I discovered this need after filling in my guilds main tank spot tonight during raiding. I need to focus our other tank and need to track debuffs on him for taunt swapping. 

**Version Number**
1.3.76

**Steps to Reproduce**
Focus a target.. try to move frame/see buffs/debuffs

**Expected Behavior**
Should be able to move focus frame to easier spot and view debuffs/buffs l

**Actual Behavior**
Can't relocate the focus frame

**Only DiabolicUI Enabled**
Yes, it is reproducable

**Additional Information**
If the frame could at least be movable I can manage buffs/debuffs on my own with other addons. In general I would love to see better built in buff/debuff support on target frames and focus frames, as these are important for raiding and doing other non-trivial tasks.

**Error Log**
N/A

**Comment by goldpaw** [comment]
Yes. I'll implement this. Working on a new focus frame currently, with more auras visible and all. The current one is pretty much useless and just in the way. 👍
